app/views/hyku_addons/account_settings/index.html.erb

```diff
@@ -10,41 +10,59 @@
   <tbody>
     <% HykuAddons::AccountBehavior::TEXT_SETTINGS.each do |fname| %>
       <tr>
-        <td id="link-<%= fname %>"><b><%= fname.humanize %>:</b> <%= @account&.settings[fname] %></td>
-        <td><%= link_to "Edit #{fname.humanize}", hyku_addons.edit_admin_account_setting_path(id: @account, partial_name: 'render_text_fields', field_name: fname), class: "btn btn-xs btn-default", remote: true %></td>
+        <% title_key = "settings.titles.#{fname}" %>
```

Comment:
This has been repeated almost 4 times:

            <% title_key = "settings.titles.#{fname}" %>

I know it is called from within different each block but do you think that using a helper will clean it up.

Reply:
You are absolute right Ed. I've moved the logic to the Helper.